packages/react/src/primitives/shared/utils.ts

```diff
@@ -34,7 +34,7 @@ export const getConsecutiveIntArray = (
   return Array.from({ length }, (_, idx) => idx + start);
 };
 
-export type Modifiers = string | number | null;
+export type Modifiers = string | number | null | undefined;
```

Comment:
I think you can drop my suggestion to add `undefined` since you have `modifiers?` below